zcash_keys/src/encoding.rs

```diff
@@ -7,17 +7,17 @@ use crate::address::UnifiedAddress;
 use bech32::primitives::decode::CheckedHrpstringError;
 use bs58::{self, decode::Error as Bs58Error};
 use std::fmt;
-use zcash_primitives::consensus::NetworkConstants;
+use zcash_protocol::consensus::{self, NetworkConstants};
 
 use zcash_address::unified::{self, Encoding};
-use zcash_primitives::{consensus, legacy::TransparentAddress};
+use zcash_transparent::address::TransparentAddress;
```

Comment:
```suggestion
use transparent::address::TransparentAddress;
```
(etc for rest of PR)